latest_tweets/utils.py

```diff
@@ -148,11 +148,26 @@ def update_tweets(messages, tweet_entities=tweet_html_entities, download=False):
             'created': tweet_created,
         })
 
-        # Add hashtags
-        tweet_hashtags(obj=obj, hashtags=i['entities'].get('hashtags', []))
+        if created:
+            # Add hashtags
+            tweet_hashtags(tweet=obj, hashtags=i['entities'].get('hashtags', []))
```

Comment:
Next time instead if naming loop variable as `i` use proper variable naming like `message` makes more sense instead of being lazy and just using `i`